src/rtc.py

```diff
@@ -166,7 +168,7 @@ def process_granule(self, local_file):
             range_looks = 12
             local_file = gpt(local_file, "TOPSAR-Deburst")
 
-        local_file = gpt(local_file, "Speckle-Filter")
+        local_file = gpt(local_file, "Speckle-Filter", "-Pfilter=Lee Sigma")
         local_file = gpt(local_file, "Multilook", f"-PnRgLooks={range_looks}", "-PnAzLooks=3")
         terrain_flattening_file = gpt(local_file, "Terrain-Flattening", "-PreGridMethod=False", "-PdemName=External DEM", f"-PexternalDEMFile={self.dem_file}", "-PexternalDEMNoDataValue=-32767")
```

Comment:
works as written, but this should use `dem_parameters` as well:

`terrain_flattening_file = gpt(local_file, "Terrain-Flattening", "-PreGridMethod=False", dem_parameters=self.dem_parameters)`